substrate/frame/root-testing/src/lib.rs

```diff
@@ -50,5 +59,14 @@ pub mod pallet {
 			ensure_root(origin)?;
 			Ok(())
 		}
+
+		#[pallet::call_index(1)]
+		#[pallet::weight(0)]
+		pub fn trigger_defensive(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin)?;
+			frame_support::defensive!("root_testing::trigger_defensive was called.");
```

Comment:
@ggwpez so you want this to panic? I don't really get what you want to achieve?

Reply:
defensive is just log error and debug_assert, which is nop on release build, so this is a bit pointless. if you want to test log error, just log error.

Reply:
Yeah I know that this is using a debug assert, I was assuming @ggwpez wants to build with debug asserts enabled and then use it as overwrite.

Reply:
The idea here is that we are currently not sure if the monitoring correctly picks up defensive failures. And we really should be notified about them, so this is just as test.  
It could also be logged, but the `defensive!` macro also logs a magic string that we can filter for and alert in the grafana.